binder/views.py

```diff
@@ -1239,13 +1330,21 @@ def get_queryset(self, request):
 
 
 
-	def order_by(self, queryset, request):
+	def _order_by_base(self, queryset, request, annotations):
 		#### order_by
 		order_bys = list(filter(None, request.GET.get('order_by', '').split(',')))
 
 		orders = []
 		if order_bys:
 			for o in order_bys:
+				head = re.match(r'^-?(.*?)(__nulls_last|__nulls_first)?$', o).group(1)
```

Comment:
Maybe make this regex a bit more explicit/extensive. With the purpose of,should we in the future want to support more special ordering operators rather then just null_last or null_first. DK how many of those there are though tbh, but would likely improve code readability a bit as well.

Reply:
AFAIK there aren't any others. And if there are just adding another one to the existing alternations should be a reasonable simple change imo. Would adding a comment to this regex suffice for you as well?

Reply:
Yeah the comment made it more clear, good enough for now and if more options need to be added this could then be changed.